### ISSUE TITLE
Remove contributors tag from docs

### ISSUE THREAD

**Issue by ShubhankarKG** [CircuitVerse/CircuitVerseDocs#230]
#### I am submitting a -
* [x] **Feature Request**
* [] **Bug Report**

#### Describe your feature -
Some pages have mentions while some pages don't. It's better to keep one format.

#### Current behaviour -
As mentioned above, some pages have mentions while some pages don't

#### Expected behaviour -
All pages shouldn't have tags like these. Since everybody must have contributed, we can't mention every person on the page. What we can do instead is create a contributions page and show the contribution graph where every contribution is mentioned.

Something like this : 

![Screenshot from 2020-05-22 11-16-24](https://user-images.githubusercontent.com/50791000/82635395-b271ab80-9c1d-11ea-8d99-5c0aab61ec88.png)

#### Screenshots (if relevant) -
![Screenshot from 2020-05-22 11-06-14](https://user-images.githubusercontent.com/50791000/82635414-c0bfc780-9c1d-11ea-89f7-5f235d54cfcc.png)
![Screenshot from 2020-05-22 11-06-17](https://user-images.githubusercontent.com/50791000/82635421-c2898b00-9c1d-11ea-8ad7-d934361fe391.png)

#### Are you working on this issue? (Yes/No)
Yes, if approved.

**Comment by Shivansh2407** [comment]
I guess it would be better if we can mention each and every contributor who has contributed to the Docs. This will motivate them a lot. For the things wherein the contributors aren't mentioned, we can look out for the commits regarding them and then mention the contributor name/ID.
Let me know your views.

**Comment by YashKumarVerma** [comment]
I think seeing the number of contributions, (even a type fix = contribution), we cannot have every name on the documentation, as it will fill up the docs with user handles. 

But, instead something like this, as is done for the interactive book can be done.
![image](https://user-images.githubusercontent.com/14032427/82637200-e5b63980-9c21-11ea-99af-960d47aec689.png)


This solution is 
- dynamic
- does not crowd the documentation
- no manual work needed (addition)

**Comment by Shivansh2407** [comment]
Yeah this can be implemented @YashKumarVerma . But what I was talking about was regarding Contributing new material like adding flip flop conversions which is a new section. So for that we can mention the name of the contributor, while for a typo fix, rephrasing content we can have the contributor section as you suggested.

**Comment by ShubhankarKG** [comment]
As far as contributing section is concerned, all contributions are covered in the contribution graph itself. Mentioning specific people might look enticing, but can also be detrimental in the sense, that for a page that had 4 or 5 contributors, only 1 had a mention.

**Comment by YashKumarVerma** [comment]
> Yeah this can be implemented @YashKumarVerma . But what I was talking about was regarding Contributing new material like adding flip flop conversions which is a new section. So for that we can mention the name of the contributor, while for a typo fix, rephrasing content we can have the contributor section as you suggested.

I don't think we should do something like that, because, at some point in future, there would a doubt regarding our definition of "new material". It can be adding a new example as well, or adding a new para of explanation about some topic, and then we'd arrive at conflicts again.

**Comment by Shivansh2407** [comment]
Ok cool. @ShubhankarKG then you can go ahead and fix this by removing all the contributors from docs and adding a contributors Section 👍

**Comment by YashKumarVerma** [comment]
> Ok cool. @ShubhankarKG then you can go ahead and fix this by removing all the contributors from docs and adding a contributors Section

or should we wait for opinions of other contributions till evening (IST)?

**Comment by ShubhankarKG** [comment]
I'll keep the PR ready just in case no one responds. Let's wait till 4 p.m.

**Comment by Shivansh2407** [comment]
> > Ok cool. @ShubhankarKG then you can go ahead and fix this by removing all the contributors from docs and adding a contributors Section
> 
> or should we wait for opinions of other contributions till evening (IST)?

Sure 👍. @satu0king @tachyons @nitin10s please let us know your views.

**Comment by ShubhankarKG** [comment]
Should we wait @YashKumarVerma , @Shivansh2407 ?

**Comment by YashKumarVerma** [comment]
> Should we wait @YashKumarVerma , @Shivansh2407 ?

Send the PR, just in case there'd be a problem, we can revert it as well.

**Comment by ShubhankarKG** [comment]
Done

**Comment by nitin10s** [comment]
Looks like a nice idea to me @satu0king. The PR needs some changes though.

**Comment by YashKumarVerma** [comment]
@nitin10s could you help us out in getting this done ?

**Comment by ShubhankarKG** [comment]
#233 has been approved by him I guess. Requesting @tachyons @satu0king to have a look

**Comment by Yana-do-code** [comment]
Closing as outdated .
